packages/slice-machine/src/components/Card/Card.tsx

```diff
@@ -149,26 +150,33 @@ export const CardFooter: FC<CardFooterProps> = ({
   startIcon,
   subtitle,
   title,
+  error = false,
   ...otherProps
-}) => (
-  <div {...otherProps} className={styles.footer}>
-    {(startIcon || loading) && (
-      <div className={styles.startIconBox}>
-        {loading && <ProgressCircle color="grey11" />}
-        {!loading && startIcon && <Icon name={startIcon} size="small" />}
+}) => {
+  const highlightColor = error ? "tomato11" : "grey11";
+
+  return (
+    <div {...otherProps} className={styles.footer}>
```

Comment:
i wonder what are these `otherProps` and if we can get rid of them, in general it is a bad idea to pass down undeclared props because typescript won't complain when we pass more things and it can affect the style and behavior of the component; consumers can rely on it and then it'll be more difficult to control

Reply:
I agree with you that we shouldn't pass down undeclared props. This was there before, so I left it without investigating. Do you think this is important enough for me to spend time looking into now?

Reply:
maybe for the end of the week and in a separate pr if no goals for the team are compromised?

Reply:
Sounds good.